### PR TITLE
feat(demo): rename some menus and add introduction doc for headless a…

### LINKED ISSUE #713: [Documentation] enhance our headless documentation
**Description**
Version 0.3 brings an entire new section to the demo website to showcase usage of our headless components with DaisyUI + their apis.

**Proposal**
Add an introduction page in the current "Components" section, rename it to "Bootstrap components", rename the section "DaisyUI" to "Headless components" and make the documentation generally clearer on our separation of packages.

### PR DESCRIPTION
**Pull request by ExFlo**
…nd BS components
closes #713

### CODE REVIEW

File: demo/src/lib/layout/Header.svelte [hunk 1]
```diff
@@ -2,10 +2,18 @@
 	import {pathToRoot$, selectedFramework$, selectedTabName$} from '../stores';
 	import {page} from '$app/stores';
 	import {getTitle} from '../../app';
+	import Svg from '$lib/layout/Svg.svelte';
+	import angularLogo from '$resources/logo-angular.svg?raw';
+	import reactLogo from '$resources/logo-react.svg?raw';
+	import svelteLogo from '$resources/logo-svelte.svg?raw';
+	import daisyUILogo from '$resources/logo-daisyUI.svg?raw';
+	import bootstrapLogo from '$resources/logo-bootstrap.svg?raw';
```
Comment (quentinderoubaix) [inline demo/src/lib/layout/Header.svelte:10]:
```suggestion
	import bootstrapLogo from 'bootstrap-icons/icons/bootstrap-fill.svg?raw';
```

File: demo/src/resources/logo-bootstrap.svg
Comment (quentinderoubaix) [inline demo/src/resources/logo-bootstrap.svg:1]: If not used in a markdown file, you can remove this and use the bootstrap svg logo provided from bootstrap-icons.

File: demo/src/lib/layout/Header.svelte [hunk 2]
```diff
@@ -24,11 +32,23 @@
...
 		{#if status === 'beta'}<span class="col-auto badge text-bg-info">Beta</span>{/if}
+		{#if selectedFramework === 'react'}<span class="col-auto d-block d-md-none p-0"><Svg svg={reactLogo} className="pres-card-logo d-flex" /></span
+			>{/if}
+		{#if selectedFramework === 'angular'}<span class="col-auto d-block d-md-none p-0"
+				><Svg svg={angularLogo} className="pres-card-logo d-flex" /></span
+			>{/if}
+		{#if selectedFramework === 'svelte'}<span class="col-auto d-block d-md-none p-0"><Svg svg={svelteLogo} className="pres-card-logo d-flex" /></span
+			>{/if}
+		{#if cssFramework === 'Bootstrap'}<span class="col-auto d-block d-md-none p-0"
+				><Svg svg={bootstrapLogo} className="pres-card-logo d-flex logo-bootstrap" /></span
```
[the remark this answers is not in the document]
Comment (quentinderoubaix) [reply demo/src/lib/layout/Header.svelte:48]: The cool AgnosUI logo of the header intersects weirdly with the display of the icons.
This is fixed by putting position-relative, but maybe there is a better way to this...

File: demo/src/resources/logo-bootstrap.svg
Comment (ExFlo) [reply demo/src/resources/logo-bootstrap.svg:1]: done

File: demo/src/lib/layout/Header.svelte (hunk 1 above)
Comment (ExFlo) [reply demo/src/lib/layout/Header.svelte:10]: done

**Review by quentinderoubaix — APPROVED** [review]
LGTM !

File: demo/src/lib/layout/Header.svelte (hunk 2 above)
[the remark this answers is not in the document]
Comment (ExFlo) [reply demo/src/lib/layout/Header.svelte]: done

**Pull request merged**